### DISCUSSION TITLE
I have two github accounts associated with an email address, but I can only see one when I log in with the email address, but I can’t find the other one. I want to delete the other unused account

### DISCUSSION THREAD

**Discussion by mamtou** [community/community#41584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

nremid

**Comment by radar-blip** [comment]
You mean you forgot the username of the other account?

You could try asking [Support](https://support.github.com/contact/account) for help. If you do, include email for them.

**Reply by mamtou** [reply]
I know the account number, but my email is not fully associated with this account, and I cannot receive the verification code

**Reply by mamtou** [reply]
https://github.com/contact/account opens 404

**Reply by radar-blip** [reply]
Its `https://support.github.com/contact/account`
